src/Stratis.Bitcoin.Tests.Common/TestBase.cs

```diff
@@ -25,31 +25,10 @@ public TestBase(Network network)
 
         public static string AssureEmptyDir(string dir)
         {
-            int deleteAttempts = 0;
-            while (deleteAttempts < 50)
-            {
-                if (Directory.Exists(dir))
-                {
-                    try
-                    {
-                        Directory.Delete(dir, true);
-                        break;
-                    }
-                    catch
-                    {
-                        deleteAttempts++;
-                        Thread.Sleep(200);
-                    }
-                }
-                else
-                    break;
-            }
-
-            if (deleteAttempts >= 50)
-                throw new Exception(string.Format("The test folder: {0} could not be deleted.", dir));
-
-            Directory.CreateDirectory(dir);
-            return dir;
+            long ticks = DateTime.UtcNow.Ticks;
```

Comment:
This seems like it is going to get really messy; how do you suggest we ensure the created directories don't accumulate forever? On the CI I assume the folders for each test run are isolated from each other regardless, so this is mostly for local tests?

Reply:
2 use cases:
1) you run tests on server after build, according to @bokobza you always start with empty dir, that way its not messy
2) you run locally, we will need to housekeep directories, I found a lot of problems when run locally and trying to delete dirs from C# this way you are guaranteed new folder every time